lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -323,6 +351,10 @@ def _on_certificate_expiring(self, event: CertificateExpiringEvent) -> None:
                             "$id": "#/properties/certificates/items/chain/items",
                         },
                     },
+                    "revoked": {
```

Comment:
If we change the schema, we should also update the [charm-relation-interfaces](https://github.com/canonical/charm-relation-interfaces) project

Reply:
Good point, here is the PR: https://github.com/canonical/charm-relation-interfaces/pull/35

Reply:
I approved it there but I'm not an admin on the project. @simskij is one of them, not sure who are the other ones.